.github/workflows/test-pip-cpu.yml

```diff
@@ -0,0 +1,36 @@
+name: Unit-tests for Pip install
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+
+  workflow_dispatch:
+
+env:
+  CHANNEL: "nightly"
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python_version: ["3.6"]
+        pytorch_args: ["", "-n", "-v 1.6", "-v 1.7", " -v 1.8", "-v 1.9"]
+      fail-fast: false
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.12xlarge
+      repository: pytorch/captum
```

Comment:
You can specify which docker image to run the job in using the `docker-image:` argument here. Feel free to use the CircleCI image that you were using before - as long as it's public, we should be able to use it here. Hopefully this makes the migration smoother